docs/UserGuide.md

```diff
@@ -45,6 +44,9 @@ Notes about the command format:
 e.g. in `add n/NAME`, NAME is a parameter which can be used as: `add n/Assignment1`.
 * Parameters can be in any order.
 e.g. if the command specifies `n/NAME t/TIME`, `t/TIME n/NAME` is also acceptable.
+* Parameters cannot contain any `/`.
```

Comment:
Maybe say this in a more formal manner? Something like `Ensure your parameter cannot contain /`. Maybe we should rename "parameter" to a simpler word like "detail" later on